backend/apps/cache/audit_middleware.py

```
@@ -0,0 +1,109 @@
+                return part
+        return None
+
+    def _get_request_data(self, request):
+        """Get request data for logging."""
+        if request.method in ['POST', 'PUT', 'PATCH']:
+            try:
+                if request.content_type == 'application/json':
+                    return json.loads(request.body.decode('utf-8'))
+                elif request.content_type == 'multipart/form-data':
+                    return {k: v for k, v in request.POST.items()}
+            except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)